src/app/(home)/changePassword/page.tsx

```diff
@@ -0,0 +1,121 @@
+import { QuestionIcon } from "../../../assets/pages/changePassword/QuestionIcon";
+import { EmailIcon } from "../../../assets/pages/changePassword/EmailIcon";
+
+import { OvalIcon } from "../../../assets/pages/changePassword/OvalIcon";
+import { ManIcon } from "../../../assets/pages/changePassword/ManIcon";
+import { VisibilityOffIcon } from "../../../assets/pages/changePassword/VisibilityOffIcon";
+
+import { ChangePasswordModal } from "../../../components/appResponse/changePasswordModal/changePasswordModal";
+
+import style from "./changePassword.module.scss";
+
+export interface IChangePassword {
+	enterNewPassword: string;
+	reenterNewPassword: string;
+}
+
+export default function ChangePassword() {
+	const {
+		register,
+		handleSubmit,
+		reset,
+		watch,
+		formState: { errors },
+	} = useForm<IChangePassword>({ mode: "onBlur" });
+
+	const onSubmit = (data: IChangePassword) => {
+		alert(JSON.stringify(data));
+		handleChangePasswordModal();
+		// changePasswordRequest;
+		reset();
+	};
+
+	const [isEnterNewPasswordShown, setIsEnterNewPasswordShown] = useState(false);
+	const [isReenterNewPasswordShown, setIsReenterNewPasswordShown] = useState(false);
+	const [isChangePasswordModalShown, setIsChangePasswordModalShown] = useState(false);
+
+	const toggleEnterPasswordVisibility = () => {
+		setIsEnterNewPasswordShown(isEnterNewPasswordShown ? false : true);
+	};
+
+	const toggleReenterPasswordVisibility = () => {
+		setIsReenterNewPasswordShown(isReenterNewPasswordShown ? false : true);
+	};
+
+	const handleChangePasswordModal = () => {
+		setIsChangePasswordModalShown(true)
+	}
+
+	return (
+		<div className={style.changePasswordWrap}>
+			<div className={style.changePasswordContainer}>
+				<OvalIcon classNames={style.ovalIcon} />
+				<QuestionIcon classNames={style.questionIcon} />
+				<EmailIcon classNames={style.emailIcon} />
+				<ArrowsIcon classNames={style.arrowsIcon} />
+				<LetterIcon classNames={style.letterIcon} />
+				<OpenLetterIcon classNames={style.openLetterIcon} />
+				<ManIcon classNames={style.manIcon} />
+				<PaperAirLineIcon classNames={style.paperAirLineIcon} />
+				<OpenLetterIcon classNames={style.secondOpenLetterIcon} />
+				<div className={style.changePasswordContainer__modal}>
+					<div className={style.changePasswordContainer__modal__content}>
+						<h1 className={style.changePasswordContainer__form__title}>Изменение пароля</h1>
+						<form onSubmit={handleSubmit(onSubmit)}>
+							{isChangePasswordModalShown && <ChangePasswordModal/>}
+							<label className={style.label}>
+								Введите новый пароль
+								<input
+									type={isEnterNewPasswordShown ? "text" : "password"}
+									className={style.changePasswordRow}
+									placeholder="Пароль"
+									{...register("enterNewPassword", {
+										pattern: /[A-Za-z]{3}/,
+									})}
+								/>
+								<VisibilityOffIcon classNames={style.visibilityOffIcon} cb={toggleEnterPasswordVisibility} />
+							</label>
```

Comment:
label htmlFor="example"
input id="example"
put input into a label is bad practice
can you change it here and in other places?